client/src/Components/Questions-Answers/AnswerModal.jsx

```diff
@@ -1,7 +1,21 @@
 import React from 'react';
 
 class AnswerModal extends React.Component {
-
+  constructor(props) {
+    super(props),
+    this.state = {
+      name: '',
+      email: '',
+      answer: ''
+    }
```

Comment:
Should each of these components be stateful or should they take props if something else already has the answer? If this was just to fill stuff in that's fine too, or if they all have varying data.